backend/infra/deploy.sh

```
@@ -0,0 +1,39 @@
+APP_HOME="/home/ubuntu/app"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)